ruby_test.rb

```diff
@@ -0,0 +1,10 @@
+puts "one"
+puts 'two'
+puts 'foo'
+
+foo = "hello"
+bar = "world"
+puts foo +  " " + bar
+
+puts foo +  " " + bar
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.